### ISSUE TITLE
Release of fine-tuned checkpionts

### ISSUE THREAD

**Issue by yui-ishihara** [marcotcr/checklist#106]
Hi,

Thank you for this great tool! 

Just wondering if you have considered releasing your fine-tuned checkpoints or training scripts (for BERT and RoBERTa) so I could reproduce your experiments (Table 1 through 3 in your original paper).

Thank you.
Yui

**Comment by marcotcr** [comment]
For SQuAD (Table 3), I used [this model](https://huggingface.co/bert-large-cased-whole-word-masking-finetuned-squad).

I used the standard huggingface example to train models on GLUE at the time, I'm sure that script is different now. I don't have the checkpoints anymore. I did release the predictions of those models [here](https://github.com/marcotcr/checklist/blob/master/release_data.tar.gz), so you can run the suites I used for the tables on them and compare to any new models on the same examples
